### ISSUE TITLE
Is the dataset for e2e missing the test and validation data?

### ISSUE THREAD

**Issue by Aaylen** [waymo-research/waymo-open-dataset#915]
All the data in waymo_open_dataset_end_to_end_camera_v_1_0_0 is labeled as training ('training_*.tfrecord*'), but the tutorial says to split the data into the three groups based on the file names:

# Replace this path with your own tfrecords.
# This tutorial is based on using data in the E2E Driving proto format directly,
# so choose the correct dataset version.
DATASET_FOLDER = '/waymo_open_dataset_'

TRAIN_FILES = os.path.join(DATASET_FOLDER, 'training.tfrecord*')
VALIDATION_FILES = os.path.join(DATASET_FOLDER, 'validation.tfrecord*')
TEST_FILES = os.path.join(DATASET_FOLDER, 'test.tfrecord*')

**Comment by nicomon24** [comment]
Hi @Aaylen, yes there is a test set coming as explain in https://waymo.com/open/data/e2e/

We are still working on this but should be coming out soon, we will send an announcement once that is done. Also notice that the submissions for the challenge will be based on this test set (which is also why currently you cannot submit a solution here https://waymo.com/open/challenges/2025/e2e-driving/).

**Comment by sanjayss34** [comment]
@nicomon24 Could you please describe what inputs are provided to the system at test time? For instance, are the numeric past positions and velocities provided, and is the intent provided?

**Comment by DerrickXuNu** [comment]
Hi @sanjayss34 , the testing data is already released. You can find what we provide from here: https://waymo.com/open/data/e2e/. Note for validation set, we also provide rater feedback labels.
